test/token_test.go

```diff
@@ -412,3 +415,105 @@ func TestMintingAndBurning(t *testing.T) {
 		executeScriptAndCheck(t, b, GenerateInspectSupplyScript(fungibleAddr, flowAddr, 1000))
 	})
 }
+
+func TestTokenAdministrator(t *testing.T) {
+	b := newEmulator()
+
+	// Should be able to deploy a contract as a new account with no keys.
+	fungibleTokenCode := readFile(fungibleTokenContractFile)
+	_, err := b.CreateAccount(nil, fungibleTokenCode)
+	assert.NoError(t, err)
+
+	_, err = b.CommitBlock()
+	assert.NoError(t, err)
+
+	// Should be able to deploy a contract as a new account with no keys.
+	flowTokenCode := readFile(flowTokenContractFile)
+
+	setupScript := fmt.Sprintf(`
+      transaction {
+        prepare(signer: AuthAccount) {
+          let flowTokenAcct = AuthAccount(payer: signer)
+
+          let admin = signer
+          flowTokenAcct.setCode("%s".decodeHex(), admin)
```

Comment:
Ideally we'd be able to do something like this, but unfortunately we can't import `FlowToken` in this transaction because it doesn't exist yet statically.

```
flowTokenAcct.setCode("%s".decodeHex())

let admin <- flowTokenAcct.load<FlowToken.Administrator>(from: /storage/flowTokenAdmin)
signer.save<FlowToken.Administrator>(<-admin, to: /storage/flowTokenAdmin)

Reply:
yeah I don't think there is any way to do it like this